iam-test-token/README.md

```diff
@@ -15,6 +15,7 @@ Once created, the JSON key should be `base64` encoded and added as secret in the
 
 ### Examples
 
+This will use the default testrunner account
```

Comment:
I would like this example to use the GCLOUD_SECRET_AUTH secret so that teams get into the habit of creating the iam-test-token-email and iam-test-token-password secrets in their own staging projects.

We should remove the global secrets and have the iam team define them in their own staging project. They are the only team that should use the current user.

Reply:
Better now? good thing is we can merge this without breaking and just let the teams know that it will stop working on friday (when we remove secrets) :)